src/main/java/de/tum/in/www1/artemis/service/scheduled/AutomaticProgrammingExerciseCleanupService.java

```diff
@@ -24,16 +23,17 @@
 import de.tum.in.www1.artemis.repository.ProgrammingExerciseStudentParticipationRepository;
 import de.tum.in.www1.artemis.security.SecurityUtils;
 import de.tum.in.www1.artemis.service.ParticipationService;
+import de.tum.in.www1.artemis.service.ProfileService;
 import de.tum.in.www1.artemis.service.connectors.GitService;
-import tech.jhipster.config.JHipsterConstants;
 
 @Service
+// NOTE: A cleanup is not necessary for LocalCI
 @Profile("scheduling")
```

Comment:
Wouldn't it make more sense to just activate the whole service only for non localci/vc applications? We also have other services that have that conditional activation.

Reply:
The git repo cleanup actually makes sense here as well, I will remove the comment here